### PR TITLE
Update locator_ros_bridge branches.

### PR DESCRIPTION
**Pull request by clalancette**
@syyuen FYI

### CODE REVIEW

File: rolling/distribution.yaml [hunk 1]
```diff
@@ -2601,7 +2601,7 @@ repositories:
     doc:
       type: git
       url: https://github.com/boschglobal/locator_ros_bridge.git
-      version: main
```
Comment (methylDragon) [inline rolling/distribution.yaml:2604]: Would these be better as https://github.com/boschglobal/locator_ros_bridge/tree/main-v1.4 ?

**Review by methylDragon — APPROVED** [review]
Just a question, otherwise looks okay.

File: rolling/distribution.yaml (hunk 1 above)
Comment (clalancette) [reply rolling/distribution.yaml:2604]: Yeah, good question.  I honestly don't know.  @syyuen will have to answer.

**Comment by clalancette** [comment]
I'm going to go ahead and merge this; if this is the wrong branch, we can update it again later.

**Pull request merged**